_templates/resources.html

```diff
@@ -17,6 +17,8 @@
   </div><div>
     <h2><img src="/img/ico_invoice.svg" class="titleicon" alt="Icon">{% translate directories %}</h2>
     <p><a href="/{{ page.lang }}/{% translate choose-your-wallet url %}">{% translate linkwallets %}</a> - bitcoin.org</p>
+    <p><a href="http://coinmap.org/">{% translate linkmap %}</a> - coinmap.org</p>
+    <p><a href="https://airbitz.co/">{% translate linkbusinessdirectory %}</a> - airbitz.co</p>
```

Comment:
@paullinator Can you use {% translate linkmerchants %} and drop the "linkmap" and "linkbusinessdirectory" entries in _translations/en.yml? These are all merchant directories, so I guess it would make sense to use existing translations.